focus-ios/Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -4585,6 +4594,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
```

Comment:
Interesting! We had to add the sim platform?

@dicarobinho 

Also tagging @isabelrios / @clarmso for automation

Reply:
@nbhasin2 more like removing mac catalyst, that translates to "iphoneos iphonesimulator".